src/test/java/core/basesyntax/service/impl/FileWriterImplTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static String data;
+    private static final String DESTINATION_FILE = "src/test/java/testfiles/testresult.txt";
+    private static final String SOURCE_FILE = "src/test/java/testfiles/SourceFileTest.txt";
+    private final FileWriterImpl fileWriter = new FileWriterImpl();
+
+    @Before
+    public void setUp() throws Exception {
+        List<String> dataTest = Files.readAllLines(Path.of(SOURCE_FILE));
+        StringBuilder stringBuilder = new StringBuilder();
+        dataTest.remove("type,fruit,quantity");
+        for (String line: dataTest) {
+            stringBuilder.append(line).append(System.lineSeparator());
+        }
+        data = stringBuilder.toString().trim();
+    }
+
+    @Test
+    public void recordDataToFile_resultFileNull_notOk() {
+        try {
+            fileWriter.write(null, data);
+        } catch (NullPointerException e) {
+            return;
+        }
+        fail();
+    }
+
+    @Test
+    public void recordDataToFile_dataNull_notOk() {
+        try {
+            fileWriter.write(DESTINATION_FILE, null);
+        } catch (NullPointerException e) {
+            return;
+        }
+        fail();
+    }
```

Comment:
```suggestion
    @Test(expected = RuntimeException.class)
    public void recordDataToFile_dataNull_notOk() {
            fileWriter.write(DESTINATION_FILE, null);
    }
```

lets make our code more consistent, you use @Test(expected = SomeException.class) in other tasts, so lets also use this here